File: VB/ImageHelper.vb
Imports System.Drawing
Imports DevExpress.LookAndFeel
Imports DevExpress.Skins
Imports DevExpress.Utils
Imports DevExpress.Utils.Drawing
Imports DevExpress.Utils.Svg

Namespace DxSample

    Public Module ImageHelper

        Public Function CreateImage(ByVal data As Byte(), ByVal Optional skinProvider As ISkinProvider = Nothing) As Image
            Dim svgBitmap As SvgBitmap = New SvgBitmap(data)
            Return svgBitmap.Render(SvgPaletteHelper.GetSvgPalette(If(skinProvider, UserLookAndFeel.Default), ObjectState.Normal), ScaleUtils.GetScaleFactor().Height)
        End Function
    End Module
End Namespace

File: CS/ImageHelper.cs
using System;
using System.Collections.Generic;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using DevExpress.LookAndFeel;
using DevExpress.Skins;
using DevExpress.Utils;
using DevExpress.Utils.Drawing;
using DevExpress.Utils.Svg;

namespace DxSample {
    public static class ImageHelper {
        public static Image CreateImage(byte[] data, ISkinProvider skinProvider = null) {
            SvgBitmap svgBitmap = new SvgBitmap(data);
            return svgBitmap.Render(SvgPaletteHelper.GetSvgPalette(skinProvider ?? UserLookAndFeel.Default, ObjectState.Normal), ScaleUtils.GetScaleFactor().Height);
        }
    }
}
